src/apps/fee-list/FeeList.entry.tsx

```diff
@@ -62,8 +67,21 @@ const FeeListEntry: FC<
     BlockedPatronEntryTextProps &
     IntermedateListEntryConfigProps &
     GlobalEntryTextProps &
-    GlobalUrlEntryPropsInterface
-> = () => <FeeList />;
+    GlobalUrlEntryPropsInterface &
+    GroupModalProps &
+    GroupModalLoansProps &
+    MaterialDetailsModalProps &
+    RenewalArgs
+> = ({ pageSizeDesktop, pageSizeMobile }) => {
+  const pageSize = pageSizeGlobal(
+    {
+      desktop: pageSizeDesktop,
+      mobile: pageSizeMobile
+    },
+    "pageSizeLoanList"
+  );
+  return <FeeList pageSize={pageSize} />;
```

Comment:
Would be nice with a comment here.
You are writing `pageSizeLoanList ` although we are dealing with a fee list?